packages/ui/src/utils/utilsDate.ts

```diff
@@ -89,3 +89,19 @@ export const formatDateFromTimestamp = (unixTime: number) => formatDate(new Date
  * @returns The adjusted timestamp in seconds, accounting for the local timezone offset
  */
 export const convertToLocaleTimestamp = (unixTime: number) => unixTime - new Date().getTimezoneOffset() * 60
+
+/**
+ * Formats a Unix timestamp into a human-readable date string adjusted for the local timezone.
+ * @param unixTime - The Unix timestamp in seconds
+ * @returns A formatted date string adjusted for the local timezone
+ */
+export const formatLocaleDateFromTimestamp = (unixTime: number) =>
+  formatDateFromTimestamp(convertToLocaleTimestamp(unixTime))
+
+/**
+ * Formats a Unix timestamp in milliseconds into a human-readable date string adjusted for the local timezone.
+ *
+ * @param unixTimeMs - The Unix timestamp in milliseconds. If wished this can be easily expanded to accept Date or string types.
+ * @returns A formatted date string adjusted for the local timezone
+ */
+export const formatLocaleDate = (unixTimeMs: number) => formatLocaleDateFromTimestamp(unixTimeMs / 1000)
```

Comment:
Imo it's become even more confusing now

This function divides the timestamp by 1000, only to multiple by 1000 again in `formatDateFromTimestamp`

Reply:
indeed, I didn't take the time to improve this. It simply replaces the old code
`formatDateFromTimestamp(convertToLocaleTimestamp(new Date(lock.timestamp).getTime() / 1000))`
as `formatLocaleDateFromTimestamp(...) = formatDateFromTimestamp(convertToLocaleTimestamp(...))` this is just the same code.
However it can be simplified to `formatDate(unixTimeMs - new Date().getTimezoneOffset() * 60000)`